### ISSUE TITLE
pikaur crashed because of KeyError

### ISSUE THREAD

**Issue by random2324** [actionless/pikaur#509]
For some reason pikaur crashed for me, first because of ttf-ancient-fonts and then on Downgrade. As I tried running with verbose it suddenly worked. 

pikaur -Vq
Pikaur v1.6.11.r0.g8ae4c56
Pacman v5.2.2 - libalpm v12.0.2

##### Attached log:

[greg@greg-pc ~]$ LC_ALL=C pikaur -Syu
:: Synchronizing package databases...
 testing is up to date
 core is up to date
 extra is up to date
 community-testing is up to date
 community is up to date
 multilib-testing is up to date
 multilib is up to date
 repo-ck is up to date
 mesa-git is up to date

:: Starting full AUR upgrade...
Reading repository package databases...
Reading local package database...
Reading AUR packages info...
:: warning: Following packages cannot be found in AUR:
    amdgpu-pro-vulkan-only    
    antergos-alerts    
    antergos-alpm-hooks    
    antergos-keyring    
    antergos-mirrorlist    
    faudio-tkg-git    
    lib32-faudio-tkg-git    
    lib32-mesa-tkg-git    
    lib32-spirv-tools-tkg-git    
    lib32-vkd3d-proton-tkg-git    
    libopenssl-1.0-compat    
    linux-amd-staging    
    linux52-tkg-pds-headers    
    linux54-tkg-pds    
    linux54-tkg-pds-headers    
    mesa-tkg-git    
    pcmciautils    
    python2-oauthlib    
    python2-pillow    
    python2-requests-oauthlib    
    rpcs3-git-bin    
    spirv-tools-tkg-git    
    vkd3d-proton-tkg-git    
    vulkan-headers-tkg-git    
    wine-tkg-git-protonified    
    xorg-font-utils    
:: Ignoring package update linux-amd-staging-drm-next-git (5.3.841339.865b4ca43816-1 => 5.6.891392.6302a40c6dfa-1)
Resolving AUR dependencies...

:: New dependencies will be installed from repository:
 community/ruby-irb (for ruby)                              -> 1.2.4-1             0.05 MiB
 community/ruby-reline (for ruby-irb)                       -> 0.1.4-1             0.03 MiB
 extra/gperf (for webkitgtk2)                               -> 3.1-3               0.10 MiB
 extra/ruby (for rubygems, ruby-reline, ruby-irb, libselinux, webkitgtk2)    -> 2.7.1-2 3.28 MiB
 extra/rubygems (for ruby)                                  -> 3.1.4-1             0.22 MiB
 extra/w3m (for vivaldi)                                    -> 0.5.3.git20200507-1 0.94 MiB

:: AUR packages will be installed:
 downgrade                             8.0.0-1              -> 8.1.0-1
 gpu-viewer                            1.23-1               -> 1.25-1
 inxi                                  3.1.03-1             -> 3.1.04-1 [outofdate: 2020/07/27]
 lib32-ffmpeg                          1:4.2.3-3            -> 2:4.3.1-1
 lib32-jasper                          2.0.16-3             -> 2.0.17-1
 lib32-libffmpeg                       1:4.2.3-3            -> 2:4.3.1-1
 lib32-vmaf                            1.5.1-1              -> 1.5.2-1
 libselinux                            3.0-1                -> 3.1-1
 libsepol                              3.0-1                -> 3.1-1
 modprobed-db                          2.43-1               -> 2.44-1
 phoronix-test-suite-milestone         9.6.0m1-1            -> 9.8.0m1-1
 plymouth-git                          0.9.4.r269.g23afdeb-1  -> 0.9.4.r278.g60c8359-1
 renderdoc-git                         1.8+242.gf238428d81-1  -> 1.9-2
 ttf-ancient-fonts                     2.60-1               -> 2.60-2
 ttf-monaco                            6.1-3                -> 6.1-6
 ttf-vista-fonts                       1-9                  -> 1-10
 vivaldi                               3.1.1929.45-1        -> 3.2.1967.41-1
 webkitgtk2                            3:2.4.11-18          -> 3:2.4.11-19

:: New dependency will be installed from AUR:
 lib32-libvmaf (for lib32-libffmpeg, lib32-vmaf)            -> 1.5.2-1

:: Proceed with installation? [Y/n] 
:: [v]iew package details   [m]anually select packages
>> y
Can't pull 'downgrade' in '/home/greg/.local/share/pikaur/aur_repos/downgrade' from AUR:

warning: Pulling without specifying how to reconcile divergent branches is
discouraged. You can squelch this message by running one of the following
commands sometime before your next pull:

  git config pull.rebase false  # merge (the default strategy)
  git config pull.rebase true   # rebase
  git config pull.ff only       # fast-forward only

You can replace "git config" with "git config --global" to set a default
preference for all repositories. You can also pass --rebase, --no-rebase,
or --ff-only on the command line to override the configured default per
invocation.

fatal: unable to access 'https://aur.archlinux.org/downgrade.git/': Could not resolve host: aur.archlinux.org

:: Try recovering?
[c] git checkout -- '*'
[r] remove dir and clone again
[s] skip this package
[a] abort
> c
looking for conflicting AUR packages...

:: Starting full system upgrade...
 there is nothing to do
Traceback (most recent call last):
  File "/usr/bin/pikaur", line 9, in <module>
    main()
  File "/usr/lib/python3.8/site-packages/pikaur/main.py", line 437, in main
    cli_entry_point()
  File "/usr/lib/python3.8/site-packages/pikaur/main.py", line 326, in cli_entry_point
    run_with_sudo_loop(pikaur_operation)
  File "/usr/lib/python3.8/site-packages/pikaur/core.py", line 319, in run_with_sudo_loop
    raise catched_exc  # pylint: disable=raising-bad-type
  File "/usr/lib/python3.8/site-packages/pikaur/core.py", line 313, in run_with_sudo_loop
    result = main_thread.get()
  File "/usr/lib/python3.8/multiprocessing/pool.py", line 771, in get
    raise self._value
  File "/usr/lib/python3.8/multiprocessing/pool.py", line 125, in worker
    result = (True, func(*args, **kwds))
  File "/usr/lib/python3.8/site-packages/pikaur/main.py", line 135, in cli_install_packages
    InstallPackagesCLI()
  File "/usr/lib/python3.8/site-packages/pikaur/install_cli.py", line 161, in __init__
    self.main_sequence()
  File "/usr/lib/python3.8/site-packages/pikaur/install_cli.py", line 180, in main_sequence
    self.install_packages()
  File "/usr/lib/python3.8/site-packages/pikaur/install_cli.py", line 994, in install_packages
    self.build_packages()
  File "/usr/lib/python3.8/site-packages/pikaur/install_cli.py", line 822, in build_packages
    pkg_build = self.package_builds_by_name[pkg_name]
KeyError: 'downgrade'
[greg@greg-pc ~]$ LC_ALL=C pikaur -Syuv
=> GET https://www.archlinux.org/feeds/news/
=> sudo pacman --color=always --sync --refresh
:: Synchronizing package databases...
 testing is up to date
 core is up to date
 extra is up to date
 community-testing is up to date
 community is up to date
 multilib-testing is up to date
 multilib is up to date
 repo-ck is up to date
 mesa-git is up to date

:: Starting full AUR upgrade...
Reading repository package databases...
Reading local package database...
=> pacman --color=always --sync --sysupgrade --print-format %r/%n
Reading AUR packages info...
=> GET https://aur.archlinux.org/rpc/?v=5&type=info&arg[]=amdgpu-pro-vulkan-only&arg[]=android-file-transfer-linux-git&arg[]=antergos-alerts&arg[]=antergos-alpm-hooks&arg[]=antergos-keyring&arg[]=antergos-mirrorlist&arg[]=antergos-wallpapers&arg[]=apitrace-git&arg[]=art-sharp&arg[]=bcunit-cunit-compat&arg[]=canon-pixma-ip7200-printer&arg[]=downgrade&arg[]=dummy-opengl-driver-git&arg[]=faudio-tkg-git&arg[]=fluxmod-styles&arg[]=gamemode-git&arg[]=gamin&arg[]=gconf&arg[]=gconf-sharp&arg[]=gconf-sharp-peditors&arg[]=gnome-mime-data&arg[]=gnome-sharp&arg[]=gnome-vfs&arg[]=gnome-vfs-sharp&arg[]=gpu-viewer&arg[]=gputest&arg[]=icu51&arg[]=imagewriter&arg[]=inxi&arg[]=kcolorpicker&arg[]=kimageannotator&arg[]=kvantum-theme-layan-git&arg[]=layan-gtk-theme-git&arg[]=layan-kde-git&arg[]=lib32-a52dec&arg[]=lib32-aom&arg[]=lib32-apitrace-git&arg[]=lib32-bluez-libs&arg[]=lib32-bluez-plugins&arg[]=lib32-chromaprint&arg[]=lib32-dav1d&arg[]=lib32-faac&arg[]=lib32-faad2&arg[]=lib32-faudio-tkg-git&arg[]=lib32-ffmpeg&arg[]=lib32-fftw&arg[]=lib32-fltk&arg[]=lib32-gconf&arg[]=lib32-glslang&arg[]=lib32-gsm&arg[]=lib32-gssdp&arg[]=lib32-gst-libav&arg[]=lib32-gst-plugins-bad&arg[]=lib32-gst-plugins-ugly&arg[]=lib32-gtk-engine-murrine&arg[]=lib32-gtk-engines&arg[]=lib32-gupnp&arg[]=lib32-gupnp-igd&arg[]=lib32-icu51&arg[]=lib32-imagemagick&arg[]=lib32-jasper&arg[]=lib32-jbigkit&arg[]=lib32-lame&arg[]=lib32-libass&arg[]=lib32-libbluray&arg[]=lib32-libbs2b&arg[]=lib32-libcdio&arg[]=lib32-libdav1d&arg[]=lib32-libdc1394&arg[]=lib32-libdca&arg[]=lib32-libde265&arg[]=lib32-libdvdcss&arg[]=lib32-libdvdnav&arg[]=lib32-libdvdread&arg[]=lib32-libexif&arg[]=lib32-libfdk-aac&arg[]=lib32-libffmpeg&arg[]=lib32-libgme&arg[]=lib32-libheif&arg[]=lib32-libkate&arg[]=lib32-liblqr&arg[]=lib32-liblrdf&arg[]=lib32-libmms&arg[]=lib32-libmpcdec&arg[]=lib32-libmpeg2&arg[]=lib32-libnice&arg[]=lib32-libofa&arg[]=lib32-libomxil-bellagio&arg[]=lib32-libplacebo&arg[]=lib32-libraqm&arg[]=lib32-libraw&arg[]=lib32-libsidplay&arg[]=lib32-libsrtp&arg[]=lib32-libstdc%2B%2B5&arg[]=lib32-libtiger&arg[]=lib32-libusb-compat&arg[]=lib32-libwmf&arg[]=lib32-lilv&arg[]=lib32-llvm-libs-minimal-git&arg[]=lib32-llvm-minimal-git&arg[]=lib32-lv2&arg[]=lib32-mesa-tkg-git&arg[]=lib32-mjpegtools&arg[]=lib32-neon&arg[]=lib32-numactl&arg[]=lib32-opencore-amr&arg[]=lib32-openexr&arg[]=lib32-openjpeg2&arg[]=lib32-raptor&arg[]=lib32-rtmpdump&arg[]=lib32-sbc&arg[]=lib32-serd&arg[]=lib32-shaderc&arg[]=lib32-sord&arg[]=lib32-spandsp&arg[]=lib32-spirv-tools-tkg-git&arg[]=lib32-sratom&arg[]=lib32-srt&arg[]=lib32-tk&arg[]=lib32-vkd3d-proton-tkg-git&arg[]=lib32-vmaf&arg[]=lib32-webrtc-audio-processing&arg[]=lib32-wildmidi&arg[]=lib32-x264&arg[]=lib32-x265&arg[]=lib32-xvidcore&arg[]=lib32-zbar&arg[]=lib32-zvbi&arg[]=libart-lgpl&arg[]=libbonobo&arg[]=libbonoboui&arg[]=libffi6&arg[]=libgnome&arg[]=libgnome-data&arg[]=libgnome-sharp&arg[]=libgnomecanvas&arg[]=libgnomeui&arg[]=libopenssl-1.0-compat&arg[]=libselinux&arg[]=libsepol&arg[]=linux-amd-staging&arg[]=linux-amd-staging-drm-next-git&arg[]=linux-amd-staging-drm-next-git-docs&arg[]=linux-amd-staging-drm-next-git-headers&arg[]=linux52-tkg-pds-headers&arg[]=linux54-tkg-pds&arg[]=linux54-tkg-pds-headers&arg[]=llvm-libs-minimal-git&arg[]=llvm-minimal-git&arg[]=mesa-tkg-git&arg[]=modprobed-db&arg[]=moonshiner&arg[]=nfs2se-git&arg[]=ntfs-3g-fuse&arg[]=openbox-themes&arg[]=orbit2&arg[]=p7zip-gui&arg[]=pamac-aur&arg[]=pamac-tray-appindicator&arg[]=pcmciautils&arg[]=phoronix-test-suite-milestone&arg[]=php-cairo-git&arg[]=php-datastructures-git&arg[]=pikaur-git&arg[]=pinta&arg[]=pkgcacheclean&arg[]=plymouth-git&arg[]=proot&arg[]=protonfixes&arg[]=protontricks&arg[]=pygtk&arg[]=python-vdf&arg[]=python2-oauthlib&arg[]=python2-pillow&arg[]=python2-requests-oauthlib&arg[]=python2-scipy&arg[]=python2-steamodd&arg[]=python2-xlib&arg[]=pywebkitgtk&arg[]=qdirstat-git&arg[]=rbdoom3-bfg-git&arg[]=renderdoc-git&arg[]=rpcs3-git-bin&arg[]=schedtoold&arg[]=spirv-headers-git&arg[]=spirv-tools-tkg-git&arg[]=systemd-boot-pacman-hook&arg[]=ttf-amiri&arg[]=ttf-ancient-fonts&arg[]=ttf-monaco&arg[]=ttf-ms-fonts&arg[]=ttf-vista-fonts&arg[]=vivaldi&arg[]=vkd3d-proton-tkg-git&arg[]=vkmark-git&arg[]=vkquake-git&arg[]=vulkan-headers-tkg-git&arg[]=wattman-gtk-git&arg[]=webkitgtk&arg[]=webkitgtk2
=> GET https://aur.archlinux.org/rpc/?v=5&type=info&arg[]=wine-tkg-git-protonified&arg[]=winetricks-git&arg[]=xorg-font-utils
:: warning: Following packages cannot be found in AUR:
    amdgpu-pro-vulkan-only    
    antergos-alerts    
    antergos-alpm-hooks    
    antergos-keyring    
    antergos-mirrorlist    
    faudio-tkg-git    
    lib32-faudio-tkg-git    
    lib32-mesa-tkg-git    
    lib32-spirv-tools-tkg-git    
    lib32-vkd3d-proton-tkg-git    
    libopenssl-1.0-compat    
    linux-amd-staging    
    linux52-tkg-pds-headers    
    linux54-tkg-pds    
    linux54-tkg-pds-headers    
    mesa-tkg-git    
    pcmciautils    
    python2-oauthlib    
    python2-pillow    
    python2-requests-oauthlib    
    rpcs3-git-bin    
    spirv-tools-tkg-git    
    vkd3d-proton-tkg-git    
    vulkan-headers-tkg-git    
    wine-tkg-git-protonified    
    xorg-font-utils    
:: Ignoring package update linux-amd-staging-drm-next-git (5.3.841339.865b4ca43816-1 => 5.6.891392.6302a40c6dfa-1)
Resolving AUR dependencies...
=> pacman --color=always --deptest pacman-contrib
=> pacman --color=always --deptest coreutils pciutils perl procps-ng
=> pacman --color=always --deptest gtk3 python python-gobject vulkan-tools clinfo mesa-demos
=> pacman --color=always --deptest ffmpeg lib32-alsa-lib lib32-aom lib32-bzip2 lib32-fontconfig lib32-fribidi lib32-glibc lib32-gmp lib32-gnutls lib32-gsm lib32-jack lib32-lame lib32-libass lib32-libavc1394 lib32-libbluray lib32-libdav1d lib32-libdrm lib32-freetype2 lib32-libiec61883 lib32-libmodplug lib32-libomxil-bellagio lib32-libpulse lib32-libraw1394 lib32-libtheora lib32-libva lib32-libvdpau lib32-libvorbis lib32-libvpx lib32-libwebp lib32-libx11 lib32-x264>=0.159 lib32-x265>=3.3 lib32-libxcb lib32-libxext lib32-libxml2 lib32-libxv lib32-xvidcore lib32-opencore-amr lib32-openjpeg2 lib32-opus lib32-sdl2 lib32-speex lib32-srt lib32-v4l-utils lib32-xz lib32-zlib ffnvcodec-headers git lib32-ladspa yasm
=> pacman --color=always --deptest ffnvcodec-headers git lib32-ladspa yasm
=> pacman --color=always --deptest pcre pkgconf python ruby xz swig
=> pacman --color=always --deptest lib32-libjpeg jasper lib32-freeglut lib32-libxmu lib32-glu cmake
=> pacman --color=always --deptest vmaf lib32-gcc-libs lib32-libvmaf meson ninja
=> pacman --color=always --deptest kmod
=> pacman --color=always --deptest glibc flex
=> pacman --color=always --deptest libdrm pango systemd git docbook-xsl
=> pacman --color=always --deptest php
=> pacman --color=always --deptest wayland libx11 libxcb mesa libgl qt5-base qt5-svg qt5-x11extras xcb-util-keysyms cmake ninja python
=> pacman --color=always --deptest fontforge fontconfig cabextract
=> pacman --color=always --deptest gtk3 libcups nss alsa-lib libxss ttf-font desktop-file-utils shared-mime-info hicolor-icon-theme w3m
=> pacman --color=always --deptest enchant>=2.2 geoclue2 gst-plugins-base-libs gtk2 harfbuzz-icu libgl libsecret libwebp libxslt libxt gobject-introspection gperf gtk3 mesa python2 ruby
=> pacman --color=always --sync --print-format=%% lib32-libvmaf
=> pacman --color=always --sync --print-format=%% ruby
=> pacman --color=always --sync --print-format=%% w3m
=> pacman --color=always --sync --print-format=%% ruby gperf
=> GET https://aur.archlinux.org/rpc/?v=5&type=info&arg[]=lib32-libvmaf
=> pacman --color=always --deptest vmaf lib32-gcc-libs meson ninja
=> pacman --color=always --deptest ffmpeg lib32-alsa-lib lib32-aom lib32-bzip2 lib32-fontconfig lib32-fribidi lib32-glibc lib32-gmp lib32-gnutls lib32-gsm lib32-jack lib32-lame lib32-libass lib32-libavc1394 lib32-libbluray lib32-libdav1d lib32-libdrm lib32-freetype2 lib32-libiec61883 lib32-libmodplug lib32-libomxil-bellagio lib32-libpulse lib32-libraw1394 lib32-libtheora lib32-libva lib32-libvdpau lib32-libvorbis lib32-libvpx lib32-libwebp lib32-libx11 lib32-x264>=0.159 lib32-x265>=3.3 lib32-libxcb lib32-libxext lib32-libxml2 lib32-libxv lib32-xvidcore lib32-opencore-amr lib32-openjpeg2 lib32-opus lib32-sdl2 lib32-speex lib32-srt lib32-v4l-utils lib32-vmaf lib32-xz lib32-zlib ffnvcodec-headers git lib32-ladspa yasm
=> pacman --color=always --deptest gtk3 libcups nss alsa-lib libxss ttf-font desktop-file-utils shared-mime-info hicolor-icon-theme w3m
=> pacman --color=always --deptest lib32-libffmpeg ffnvcodec-headers git lib32-ladspa yasm
=> pacman --color=always --deptest vmaf lib32-gcc-libs lib32-libvmaf meson ninja
=> pacman --color=always --deptest enchant>=2.2 geoclue2 gst-plugins-base-libs gtk2 harfbuzz-icu libgl libsecret libwebp libxslt libxt gobject-introspection gperf gtk3 mesa python2 ruby
=> pacman --color=always --deptest libsepol>=3.1 pcre pkgconf python ruby xz swig
=> pacman --color=always --sync w3m --print-format %r/%n
=> pacman --color=always --sync ruby --print-format %r/%n
=> pacman --color=always --sync gperf --print-format %r/%n
=> pacman --color=always --sync ruby --print-format %r/%n
=> pacman --color=always --sync --needed ruby ruby w3m gperf --print-format %r/%n

:: New dependencies will be installed from repository:
 community/ruby-irb (for ruby)                              -> 1.2.4-1             0.05 MiB
 community/ruby-reline (for ruby-irb)                       -> 0.1.4-1             0.03 MiB
 extra/gperf (for webkitgtk2)                               -> 3.1-3               0.10 MiB
 extra/ruby (for rubygems, ruby-reline, ruby-irb, libselinux, webkitgtk2)    -> 2.7.1-2 3.28 MiB
 extra/rubygems (for ruby)                                  -> 3.1.4-1             0.22 MiB
 extra/w3m (for vivaldi)                                    -> 0.5.3.git20200507-1 0.94 MiB

:: AUR packages will be installed:
 downgrade                             8.0.0-1              -> 8.1.0-1
 gpu-viewer                            1.23-1               -> 1.25-1
 inxi                                  3.1.03-1             -> 3.1.04-1 [outofdate: 2020/07/27]
 lib32-ffmpeg                          1:4.2.3-3            -> 2:4.3.1-1
 lib32-jasper                          2.0.16-3             -> 2.0.17-1
 lib32-libffmpeg                       1:4.2.3-3            -> 2:4.3.1-1
 lib32-vmaf                            1.5.1-1              -> 1.5.2-1
 libselinux                            3.0-1                -> 3.1-1
 libsepol                              3.0-1                -> 3.1-1
 modprobed-db                          2.43-1               -> 2.44-1
 phoronix-test-suite-milestone         9.6.0m1-1            -> 9.8.0m1-1
 plymouth-git                          0.9.4.r269.g23afdeb-1  -> 0.9.4.r278.g60c8359-1
 renderdoc-git                         1.8+242.gf238428d81-1  -> 1.9-2
 ttf-ancient-fonts                     2.60-1               -> 2.60-2
 ttf-monaco                            6.1-3                -> 6.1-6
 ttf-vista-fonts                       1-9                  -> 1-10
 vivaldi                               3.1.1929.45-1        -> 3.2.1967.41-1
 webkitgtk2                            3:2.4.11-18          -> 3:2.4.11-19

:: New dependency will be installed from AUR:
 lib32-libvmaf (for lib32-libffmpeg, lib32-vmaf)            -> 1.5.2-1

:: Proceed with installation? [Y/n] 
:: [v]iew package details   [m]anually select packages
>> y
=> git -C /home/greg/.local/share/pikaur/aur_repos/lib32-jasper pull origin master
=> git -C /home/greg/.local/share/pikaur/aur_repos/lib32-ffmpeg pull origin master
=> git -C /home/greg/.local/share/pikaur/aur_repos/inxi pull origin master
=> git -C /home/greg/.local/share/pikaur/aur_repos/downgrade pull origin master
=> git -C /home/greg/.local/share/pikaur/aur_repos/gpu-viewer pull origin master
=> git clone https://aur.archlinux.org/libsepol.git /home/greg/.local/share/pikaur/aur_repos/libsepol
=> git -C /home/greg/.local/share/pikaur/aur_repos/libselinux pull origin master
=> git -C /home/greg/.local/share/pikaur/aur_repos/lib32-vmaf pull origin master
=> git -C /home/greg/.local/share/pikaur/aur_repos/modprobed-db pull origin master
=> git -C /home/greg/.local/share/pikaur/aur_repos/phoronix-test-suite-milestone pull origin master
=> git -C /home/greg/.local/share/pikaur/aur_repos/plymouth-git pull origin master
=> git -C /home/greg/.local/share/pikaur/aur_repos/renderdoc-git pull origin master
=> git -C /home/greg/.local/share/pikaur/aur_repos/ttf-ancient-fonts pull origin master
=> git -C /home/greg/.local/share/pikaur/aur_repos/ttf-monaco pull origin master
=> git -C /home/greg/.local/share/pikaur/aur_repos/ttf-vista-fonts pull origin master
=> git -C /home/greg/.local/share/pikaur/aur_repos/vivaldi pull origin master
=> git -C /home/greg/.local/share/pikaur/aur_repos/webkitgtk2 pull origin master
=> pacman --color=always --deptest ffmpeg lib32-alsa-lib lib32-aom lib32-bzip2 lib32-fontconfig lib32-fribidi lib32-glibc lib32-gmp lib32-gnutls lib32-gsm lib32-jack lib32-lame lib32-libass lib32-libavc1394 lib32-libbluray lib32-libdav1d lib32-libdrm lib32-freetype2 lib32-libiec61883 lib32-libmodplug lib32-libomxil-bellagio lib32-libpulse lib32-libraw1394 lib32-libtheora lib32-libva lib32-libvdpau lib32-libvorbis lib32-libvpx lib32-libwebp lib32-libx11 lib32-x264>=0.159 lib32-x265>=3.3 lib32-libxcb lib32-libxext lib32-libxml2 lib32-libxv lib32-xvidcore lib32-opencore-amr lib32-openjpeg2 lib32-opus lib32-sdl2 lib32-speex lib32-srt lib32-v4l-utils lib32-vmaf lib32-xz lib32-zlib
=> pacman --color=always --deptest lib32-libjpeg jasper
=> pacman --color=always --deptest lib32-freeglut lib32-libxmu lib32-glu cmake
=> pacman --color=always --deptest vmaf lib32-gcc-libs
=> pacman --color=always --deptest meson ninja
=> pacman --color=always --deptest libsepol>=3.1 pcre
=> pacman --color=always --deptest pkgconf python ruby xz swig
=> pacman --color=always --deptest glibc
=> pacman --color=always --deptest flex
=> pacman --color=always --deptest libdrm pango systemd
=> pacman --color=always --deptest git docbook-xsl
=> pacman --color=always --deptest wayland libx11 libxcb mesa libgl qt5-base qt5-svg qt5-x11extras xcb-util-keysyms
=> pacman --color=always --deptest cmake ninja python
=> pacman --color=always --deptest gtk3 libcups nss alsa-lib libxss ttf-font desktop-file-utils shared-mime-info hicolor-icon-theme
=> pacman --color=always --deptest w3m
=> pacman --color=always --deptest enchant>=2.2 geoclue2 gst-plugins-base-libs gtk2 harfbuzz-icu libgl libsecret libwebp libxslt libxt
=> pacman --color=always --deptest gobject-introspection gperf gtk2 gtk3 mesa python2 ruby
looking for conflicting AUR packages...
=> pacman --color=always --deptest gtk3 libcups nss alsa-lib libxss ttf-font desktop-file-utils shared-mime-info hicolor-icon-theme w3m
=> pacman --color=always --deptest libsepol>=3.1 pcre pkgconf python ruby xz swig
=> pacman --color=always --deptest enchant>=2.2 geoclue2 gst-plugins-base-libs gtk2 harfbuzz-icu libgl libsecret libwebp libxslt libxt gobject-introspection gperf gtk3 mesa python2 ruby
=> pacman --color=always --deptest vmaf lib32-gcc-libs lib32-libvmaf meson ninja
Do you want to see build files diff for modprobed-db package? [Y/n] n
:: Skipping review of PKGBUILD for modprobed-db package (--noedit)
:: Skipping review of readme.install for modprobed-db package (--noedit)
:: warning: Not showing diff for lib32-jasper package (installing for the first time)
:: Skipping review of PKGBUILD for lib32-jasper package (--noedit)
:: warning: Not showing diff for ttf-ancient-fonts package (installing for the first time)
:: Skipping review of PKGBUILD for ttf-ancient-fonts package (--noedit)
Do you want to see build files diff for lib32-ffmpeg, lib32-libffmpeg package? [Y/n] n
:: Skipping review of PKGBUILD for lib32-ffmpeg, lib32-libffmpeg package (--noedit)
Do you want to see build files diff for phoronix-test-suite-milestone package? [Y/n] n
:: Skipping review of PKGBUILD for phoronix-test-suite-milestone package (--noedit)
Do you want to see build files diff for plymouth-git package? [Y/n] n
:: Skipping review of PKGBUILD for plymouth-git package (--noedit)
:: warning: Not showing diff for libsepol package (installing for the first time)
:: Skipping review of PKGBUILD for libsepol package (--noedit)
Do you want to see build files diff for downgrade package? [Y/n] n
:: Skipping review of PKGBUILD for downgrade package (--noedit)
:: warning: Not showing diff for webkitgtk2 package (installing for the first time)
:: Skipping review of PKGBUILD for webkitgtk2 package (--noedit)
:: Skipping review of webkitgtk.install for webkitgtk2 package (--noedit)
Do you want to see build files diff for renderdoc-git package? [Y/n] n
:: Skipping review of PKGBUILD for renderdoc-git package (--noedit)
:: warning: Not showing diff for ttf-monaco package (installing for the first time)
:: Skipping review of PKGBUILD for ttf-monaco package (--noedit)
:: warning: Not showing diff for gpu-viewer package (installing for the first time)
:: Skipping review of PKGBUILD for gpu-viewer package (--noedit)
Do you want to see build files diff for inxi package? [Y/n] n
:: Skipping review of PKGBUILD for inxi package (--noedit)
Do you want to see build files diff for lib32-vmaf, lib32-libvmaf package? [Y/n] n
:: Skipping review of PKGBUILD for lib32-vmaf, lib32-libvmaf package (--noedit)
Do you want to see build files diff for vivaldi package? [Y/n] n
:: Skipping review of PKGBUILD for vivaldi package (--noedit)
:: warning: Not showing diff for libselinux package (installing for the first time)
:: Skipping review of PKGBUILD for libselinux package (--noedit)
:: warning: Not showing diff for ttf-vista-fonts package (installing for the first time)
:: Skipping review of PKGBUILD for ttf-vista-fonts package (--noedit)
:: Skipping review of ttf-vista-fonts.install for ttf-vista-fonts package (--noedit)

=> sudo pacman --color=always --sync --verbose --sysupgrade
Root      : /
Conf File : /etc/pacman.conf
DB Path   : /var/lib/pacman/
Cache Dirs: /var/cache/pacman/pkg/  
Hook Dirs : /usr/share/libalpm/hooks/  /etc/pacman.d/hooks/  
Lock File : /var/lib/pacman/db.lck
Log File  : /var/log/pacman.log
GPG Dir   : /etc/pacman.d/gnupg/
Targets   : None
:: Starting full system upgrade...
 there is nothing to do
=> mkdir -p /home/greg/.cache/pikaur/build/lib32-vmaf
=> cp -r /home/greg/.local/share/pikaur/aur_repos/lib32-vmaf/last_installed.txt /home/greg/.local/share/pikaur/aur_repos/lib32-vmaf/PKGBUILD /home/greg/.local/share/pikaur/aur_repos/lib32-vmaf/.SRCINFO /home/greg/.cache/pikaur/build/lib32-vmaf/
=> pacman --color=always --deptest vmaf lib32-gcc-libs
=> pacman --color=always --deptest meson ninja
=> makepkg --packagelist

:: Starting the build:
=> makepkg --force
==> Making package: lib32-vmaf 1.5.2-1 (Wed Aug  5 11:35:29 2020)
==> Checking runtime dependencies...
==> Checking buildtime dependencies...
==> Retrieving sources...
  -> Downloading vmaf-1.5.2.tar.gz...

**Comment by random2324** [comment]
Created just in case someone else finds it useful. Closing before the author does.

**Comment by actionless** [comment]
could you please provide minimal reproducible steps sequence?

because it's so much mis-formated text what it's hard to understand what's the point here

**Comment by random2324** [comment]
Okay, I try to describe the issue as detailed as I can.

I ran pikaur -Syu in a yakuake window.
Yakuake Version 3.0.5+
KDE Frameworks 5.72.0
Qt 5.15.0 (kompiliert gegen 5.15.0)

It updated the arch repository packages just fine. But arch AUR packages failed once it tried to get the newest version from the server:
`fatal: unable to access 'https://aur.archlinux.org/downgrade.git/': Could not resolve host: aur.archlinux.org`
I have a very bad (wireless) internet connection and as I downloaded something else in the background, it probably timed out. As this happened, pikaur asked how to proceed:
```
:: Try recovering?
[c] git checkout -- '*'
[r] remove dir and clone again
[s] skip this package
[a] abort
```
I typed "c" or "r" BUT input was ignored.

```
c
looking for conflicting AUR packages...

:: Starting full system upgrade...
there is nothing to do
Traceback (most recent call last):
File "/usr/bin/pikaur", line 9, in
main()
File "/usr/lib/python3.8/site-packages/pikaur/main.py", line 437, in main
cli_entry_point()
File "/usr/lib/python3.8/site-packages/pikaur/main.py", line 326, in cli_entry_point
run_with_sudo_loop(pikaur_operation)
File "/usr/lib/python3.8/site-packages/pikaur/core.py", line 319, in run_with_sudo_loop
raise catched_exc # pylint: disable=raising-bad-type
File "/usr/lib/python3.8/site-packages/pikaur/core.py", line 313, in run_with_sudo_loop
result = main_thread.get()
File "/usr/lib/python3.8/multiprocessing/pool.py", line 771, in get
raise self._value
File "/usr/lib/python3.8/multiprocessing/pool.py", line 125, in worker
result = (True, func(*args, **kwds))
File "/usr/lib/python3.8/site-packages/pikaur/main.py", line 135, in cli_install_packages
InstallPackagesCLI()
File "/usr/lib/python3.8/site-packages/pikaur/install_cli.py", line 161, in init
self.main_sequence()
File "/usr/lib/python3.8/site-packages/pikaur/install_cli.py", line 180, in main_sequence
self.install_packages()
File "/usr/lib/python3.8/site-packages/pikaur/install_cli.py", line 994, in install_packages
self.build_packages()
File "/usr/lib/python3.8/site-packages/pikaur/install_cli.py", line 822, in build_packages
pkg_build = self.package_builds_by_name[pkg_name]
KeyError: 'downgrade'
```
As you can see, its like I entered nothing even though I did! In the next lines you can see how pikaur crashes because it can't install the packages.

**Comment by actionless** [comment]
"reproducible" doesn't mean detailed, it means what i could repeat the same actions on my pc and will be able to reproduce it there

**Comment by random2324** [comment]
You clearly see what I did above.
I said I ran `LC_ALL=C pikaur -Syu` but it failed on the AUR packages because it couldn't resolve the aur host because it probably timed out to early because of my bad internet connection. I guess you're clever enough to provoke this "could not resolve host error" by disabling your network after you see the summary of to installed aur packages so it fails in the same way. You also don't need specific packages as its more a general problem.

So thats all steps to reproduce like I already typed 3 times above. I cant provide more. Im happy to give you more details but I dont know whats important for you. So you simply has to ask specifically.

I just can give you my locale settings as I guess thats the culprit:
LANG=de_DE.UTF-8
LC_COLLATE=de_DE.UTF-8
